.github/workflows/cypress-workflow.yml

```diff
@@ -155,8 +155,7 @@ jobs:
         uses: cypress-io/github-action@v5
         with:
           working-directory: OpenSearch-Dashboards/plugins/dashboards-maps
-          command: yarn run cypress run --browser chrome
-          wait-on: 'http://localhost:5601'
+          command: yarn run cypress run --headless --browser chrome
```

Comment:
The latest version chrome browser used higher memory/CPU which crushes the CI runner, let's switch to use `electron` which worked from my CI test. Also please add note here the reason why we switched.